style.css

```diff
@@ -11,6 +11,10 @@ body {
   background-color: #e5e5e5;
 }
 
+.navbar-link {
+  display: none;
+}
+
```

Comment:
- [ ] Great job so far :raised_hands: but I think it would be better if you could improve your close (X) button to match to one provided in the original design (e.g., font, colors, images, text, margins). Please change this. This will make your website similar to the original design. 💯 🥇 

 - [ ] Nice implementing the design, however, it would be better if you add a little bit of blur to your overlay in order to match the Figma design and also make a good contrast 🎨 👨🏻‍🎨 

- [OPTIONAL] What about preventing the sub-background from scrolling when the navbar is open? You can use a CSS property `over-flow: hidden;` to do that, which will make your mobile menu correspond to the original design.💯 🥇 

-  Adding these features will make your design look great and look exactly like the Figma design, I highly recommend you do this in the meantime, I'll approve this project since the previous reviewer is skipping this.


| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/79058364/217431605-ded49b10-fcbb-41bb-888a-b460ffbf7996.png) | ![image](https://user-images.githubusercontent.com/61790587/167505757-7e8c3eba-c166-4f42-842b-829d676cabd5.png) |